src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyFulfillmentOrderPayload.java

```diff
@@ -0,0 +1,36 @@
+package com.shopify.model.fulfillmentOrderApi;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyFulfillmentOrderPayload {
+
+	@XmlElement(name = "new_location_id")
```

Comment:
Is this the payload for moving since it's called new_location_id? If it is can we rename this and have a separate payload